homeassistant/components/remember_the_milk/entity.py

```diff
@@ -60,20 +60,21 @@ def create_task(self, call: ServiceCall) -> None:
             result = self._rtm_api.rtm.timelines.create()
             timeline = result.timeline.value
 
-            if hass_id is None or rtm_id is None:
+            if rtm_id is None:
```

Comment:
This check was simplified since the first case is always true if the second case is true.